config/webhook/kustomization.yaml

```diff
@@ -20,3 +20,24 @@ patchesJson6902:
     - op: replace
       path: /webhooks/0/clientConfig/service/name
       value: k8ssandra-operator-webhook-service
+
+# adding the objectSelector prevents the bootstrapping problem
+# where the mutation request for the operator pod would be 
+# sent before the operator pod is created
+patchesJson6902:
```

Comment:
I'm still trying to get tests to run for me locally so I can debug your e2e tests which are failing. 

The observation I will make, is that there are no failures on the cluster scoped test. That leads me to wonder whether there is an issue around Rolebindings/ClusterRoleBindings which you need to address to ensure that your data plane deployments have perms to access Pods in namespaces other than their own.

I'll keep trying to get a kind cluster up with the objective of inspecting the errors as the test runs.

Reply:
yeah I'm having a bit of an issue running the e2e tests locally as well; they seem to just hang and never complete

Reply:
ah ok I think I've fixed it. I was missing a namespace patch in the test fixtures